eloquent.md

```diff
@@ -940,10 +940,6 @@ In the example above, we are retrieving the model from the database before calli
 
     Flight::destroy(collect([1, 2, 3]));
 
-If you are utilizing [soft deleting models](#soft-deleting), you may permanently delete models via the `forceDestroy` method:
-
-    Flight::forceDestroy(1);
-
```

Comment:
Why did you remove this?

Reply:
Just mistake when rebasing from master. Fixed